lib/foo.go

```diff
@@ -0,0 +1,12 @@
+package libfoo
+
+import (
+	"fmt"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+func Foo() (int, error) {
```

Comment:
__[golint]__
* [golint] exported function Foo should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->